pallets/corporate-actions/src/lib.rs

```diff
@@ -0,0 +1,288 @@
+// This file is part of the Polymesh distribution (https://github.com/PolymathNetwork/Polymesh).
+// Copyright (c) 2020 Polymath
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3.
+
+// This program is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+//! # Corporate Actions module.
+//!
+//! TODO
```

Comment:
We either need to complete these in this PR or capture in a new story.

Reply:
Filed as https://polymath.atlassian.net/browse/MESH-1336.

Reply:
Resolved in https://github.com/PolymathNetwork/Polymesh/pull/716.